crates/pessimistic-proof-core/src/bridge_exit.rs

```diff
@@ -70,18 +70,18 @@ const EMPTY_METADATA_HASH: Digest = Digest(hex!(
 
 pub fn bridge_exit_hasher(
     leaf_type: u8,
-    origin_network: u32,
+    origin_network: u16,
```

Comment:
I think that the spec defines that `networkId` are `u32`, @hadjiszs do you think we could scope this to only the `program`?

Reply:
hm I think we can't modify it there, the hash of the bridge exits should stay the same otherwise the re-computed local exit root will differ with everywhere else on the stack (L1, cdk, etc).

if we want to reduce the size taken for one `NetworkId`, it could only be done on the balance tree and the nullifier tree, because only the agglayer + pp compute those two trees

However I wonder how this can be upgraded given that the PP is already in mainnet, we'd have to create one PP which starts and finish its transition on trees with different depth, do you have some ideas for that @Nashtare ?